Frameworks/GLKit/GLKTexture.mm

```diff
@@ -414,13 +421,10 @@ + (GLKTextureInfo*)cubeMapWithContentsOfFile:(NSString*)fname options:(NSDiction
 
     if (h != 6 * w) {
         NSTraceError(TAG, @"ERROR - Unexpected cube map format, expected 6 square textures aligned vertically.");
-        CGImageRelease(img);
-        CGDataProviderRelease(provider);
         return nil;
     }
 
-    CGDataProviderRelease(provider);
-    provider = CGImageGetDataProvider(img);
+    CGDataProviderRef provider = CGImageGetDataProvider(img);
```

Comment:
Since you are fixing this code, you might as well make it use `_CGDataProviderGetBytes`.